locales/it/index.yml

```
@@ -279,6 +279,49 @@ authentication:
   expiredSessionBanner:
     title: Avviso di sicurezza
     message: Per mantenere al sicuro i tuoi dati privati, accedi nuovamente con le tue credenziali SPID, grazie!
+email:
+  read:
+    title: Il tuo indirizzo email
+    info: Questo è l'indirizzo email che hai associato all'account SPID. Puoi
+      usare in IO un indirizzo email diverso, se preferisci. Per rendere operativo
+      il nuovo indirizzo, dovrai validarlo cliccando sul link che riceverai nella tua
+      casella di posta elettronica.
+    details: Puoi modificare questo indirizzo, che dovrai validare cliccando su un  
+      link che spediremo alla tua casella di posta. Nel frattempo, le funzioni di IO che 
+      dipendono dalla email (pagamenti, inoltro messaggi, etc.) saranno disattivate.
+    alert: "Attenzione: il tuo account PagoPA è associato alla tua email,
+      quindi se la modifichi non troverai più elencati
+      nell'applicazione le tue transazioni ed i tuoi metodi di pagamento."
+    help: (onb.email.help) (onb.help.read.help) Il tuo profilo SPID trasferisce ad IO il tuo indirizzo email. Se è un
```

```suggestion
    help: (onb.email.help) (onb.help.read.help) Il tuo profilo SPID trasferisce ad IO il tuo indirizzo email. Se è un
```
(onb.email.help) (onb.help.read.help) ??

---

Removed within updated within https://github.com/teamdigitale/io-app/pull/1363/commits/2d1442691356a14fc1b5864a0a30220da184f8cb